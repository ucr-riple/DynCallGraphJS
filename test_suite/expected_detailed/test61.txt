{
    'global (test61.js:13:1:13:6)': [
        'foo (test61.js:8:1:12:2)'
    ],
    'foo (test61.js:10:9:10:16)': [
        'bar (test61.js:4:1:6:2)'
    ]
}