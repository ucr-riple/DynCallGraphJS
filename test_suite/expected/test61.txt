{ '(test61.js:13:1:13:6)':
   [ '(test61.js:8:1:12:2)' ],
  '(test61.js:10:9:10:16)':
   [ '(test61.js:4:1:6:2)' ] }